packages/stream_chat_persistence/lib/src/dao/pinned_message_dao.dart

```diff
@@ -135,7 +135,7 @@ class PinnedMessageDao extends DatabaseAccessor<MoorChatDatabase>
           pinnedMessages.pinnedByUserId.equalsExp(_pinnedByUsers.id)),
     ])
           ..where(pinnedMessages.channelCid.equals(cid))
-          ..where(isNull(pinnedMessages.parentId) |
+          ..where(pinnedMessages.parentId.isNotNull() |
```

Comment:
Consider changing this to `isNull`.

Reply:
Good catch! Need to stop coding at midnight 😛